docker/hsm.Dockerfile

```diff
@@ -1,6 +1,16 @@
 # Use the amazon linux image
 FROM amazonlinux:2
 
+# Install SoftHSM2
+RUN yum install -y softhsm
+
+# Initialize some HSM slots as initial setup to be used in tests
+RUN softhsm2-util --init-token --slot 0 --label WALLET-000 --pin us3rs3cur3 --so-pin sup3rs3cur3
+RUN softhsm2-util --init-token --slot 1 --label WALLET-001 --pin us3rs3cur3 --so-pin sup3rs3cur3
+RUN softhsm2-util --init-token --slot 2 --label WALLET-002 --pin us3rs3cur3 --so-pin sup3rs3cur3
+RUN softhsm2-util --init-token --slot 3 --label WALLET-003 --pin us3rs3cur3 --so-pin sup3rs3cur3
+RUN softhsm2-util --init-token --slot 4 --label cavium --pin us3r:s3cur3 --so-pin sup3r:s3cur3
```

Comment:
this initialization is specific fixture for some testing stuff, it shouldn't be part of the base ethsigner image. We can create anothe Dockerfile somewhere else with something like this:

```Dockerfile
# adharaprojects/ethsinger:0.7.0-ADHARA-testfixtures
FROM adharaprojects/ethsinger:0.7.0-ADHARA

RUN softhsm2-util --init-token --slot 0 --label WALLET-000 --pin us3rs3cur3 --so-pin sup3rs3cur3
RUN softhsm2-util --init-token --slot 1 --label WALLET-001 --pin us3rs3cur3 --so-pin sup3rs3cur3
RUN softhsm2-util --init-token --slot 2 --label WALLET-002 --pin us3rs3cur3 --so-pin sup3rs3cur3
RUN softhsm2-util --init-token --slot 3 --label WALLET-003 --pin us3rs3cur3 --so-pin sup3rs3cur3
RUN softhsm2-util --init-token --slot 4 --label cavium --pin us3r:s3cur3 --so-pin sup3r:s3cur3
```